### ISSUE TITLE
Improve documentation for restricted enterprise install

### ISSUE THREAD

**Issue by kjpgit** [microsoft/WSL#9575]
I work at a corp that does not allow local admin access for the main user id (which uses a domain logon).   Corp also blocks windows store.

The only admin access allowed is to a secondary, local user id.  

The solution I did, adapting from [this page ](https://www.omgubuntu.co.uk/how-to-install-wsl2-on-windows-10) was this:

1) Log in as secondary user (administrator), and run elevated command prompt "as administrator".  Run these commands in it:

dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart

dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart

wsl --set-default-version 2

2) Log back in as main user (not administrator) and run these commands:

wsl --update   # Note: unsure if this should/could be run as admin instead

wsl --install --download Ubuntu

Note that you get "Catastrophic error" if you run the simple `wsl --install` as the wrong user id, even though it's an administrator.  That's what I tried first, and [saw this ticket](https://github.com/microsoft/WSL/issues/9420) where someone had the same problem.  Thus, the need for splitting up the admin vs non-admin steps.

**Comment by vladimirgluten** [comment]
I don't think this will work for me because I'm not able to log into a PC with my admin account. I can log into a server, but not a desktop or laptop.  My admin account only works for when I get the elevated permissions prompt.

**Comment by isaac84** [comment]
I have the same issue as @vladimirgluten. I managed to install wsl2 with docker. But i can't run `wsl --update` i just get the Catastrophic Failure.

My user has 'Elevation access' so I can run the console as administrator but it doesn't help.

**Comment by istvankleijn** [comment]
Thank you, I am in the same situation and was able to install Ubuntu 22.04 using your workaround.
FWIW, I ran `wsl --update` while still logged in as my local admin.
